python/tchannel/messages/call_continue.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+
+from . import common, ChecksumType
+from .base import BaseMessage
+from .common import FlagsType
+
+
+class CallContinueMessage(BaseMessage):
+    """Represent a continuation of a call request (across multiple frames)."""
+    max_args_num = 3
+
+    __slots__ = (
+        'flags',
+        'checksum',
+        'args',
+    )
+
+    def __init__(
+        self,
+        flags=0,
+        checksum=None,
+        args=None,
+    ):
+        self.flags = flags
+        if checksum is not None:
+            checksum = common.ChecksumType.standardize(checksum)
+        self.checksum = checksum or \
+            (common.ChecksumType.none, None)
```

Comment:
Change to:

    if checksum is None:
        self.checksum = ...
    else:
        self.checksum = standardize(checksum)